Automattic/jetpack
# Add new site setting: Third party data opt-out

## Proposed changes:

Adds a new site setting option `wpcom_data_sharing_opt_out` in v1.4 /site/$site/settings endpoint, to help users have more granular control over how their data is used. 

### Other information:

The setting is only relevant to WPCOM and WoA sites at the moment. 
This setting will be toggle-able in Calypso site privacy area. 

## Jetpack product discussion



peGwbA-1px-p2#comment-2805

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






Sandbox public-api
After syncing your changes to wpcom, you can test with https://developer.wordpress.com/docs/api/console/
Update the setting with 
<img width="612" alt="image" src="https://github.com/Automattic/jetpack/assets/7129409/e7ed9a01-b50a-498e-8abf-a8b432a1b619">
And check the value with GET
